targets/GenApi.targets

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This is here to allow the enabling and disabling of the functionality.  If GenerateReferenceAssemblySources is false, then the GenApi package targets are not imported.  I could have put all of this in dirs.targets but thought it looked cleaner it it's own file.  What's your opinion?

---
In reply to: [78388700](https://github.com/Microsoft/msbuild/pull/1013#discussion_r78388700) [](ancestors = 78388700)